fusions.wdl

```diff
@@ -44,41 +40,40 @@ task samtools_fusions {
 # Samtools task for mutect
 task samtools_mutect {
     input {
-        File dnmt3a_bed
-        File full_wgs
-        File full_wgs_idx
+        File subset_bed
+        File full_cram
+        File full_cram_crai
         File ref_fa
         File ref_fa_fai
     }
     Int cores = 1
-    Float bam_size = size([full_wgs], "GB")
+    Float bam_size = size([full_cram], "GB")
     Int runtime_size = 4 + round(bam_size)
     runtime {
         memory: "4GB"
         cpu: cores
         preemptible: 1
-        docker: "chrisamiller/genomic-analysis:0.2"
+        docker: "quay.io/biocontainers/samtools:1.2--0"
         disks: "local-disk ~{runtime_size} SSD"
         bootDiskSizeGb: runtime_size
     }
     command <<<
         set -o pipefail
         set -o errexit
-        ln -s ~{full_wgs} full.wgs
-        ln -s ~{full_wgs_idx} full.wgs.idx
-        samtools view -b -L ~{dnmt3a_bed} -T ~{ref_fa} -t ~{ref_fa_fai} -o dnmt3a.bam full.wgs
-        samtools index -b dnmt3a.bam
+        ln -s ~{full_cram} full.cram
+        ln -s ~{full_cram_crai} full.cram.crai
+        samtools view -b -L ~{subset_bed} -M -T ~{ref_fa} -t ~{ref_fa_fai} -F 1036 -f 1 -o subset.bam full.cram
```

Comment:
Could there be a situation where, for example, read1 is mapped, the mate (read2) is unmapped, but there is a supplementary alignment on read1 that supports a fusion? Just trying to think of weird edge cases...